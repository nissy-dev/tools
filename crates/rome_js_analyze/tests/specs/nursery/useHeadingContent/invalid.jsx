<>
	<h1 />
	<h1>
		<TextWrapper aria-hidden />
	</h1>
	<h1>
		<div aria-hidden />
	</h1>
	<h1></h1>
</>;
