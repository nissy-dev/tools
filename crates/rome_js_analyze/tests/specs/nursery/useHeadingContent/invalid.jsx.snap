---
source: crates/rome_js_analyze/tests/spec_tests.rs
expression: invalid.jsx
---
# Input
```js
<>
	<h1 />
	<h1>
		<TextWrapper aria-hidden />
	</h1>
	<h1>
		<div aria-hidden />
	</h1>
	<h1></h1>
</>;

```

# Diagnostics
```
invalid.jsx:2:2 lint/nursery/useHeadingContent ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ! Provide screen reader accessible content when using heading  elements.
  
    1 │ <>
  > 2 │ 	<h1 />
      │ 	^^^^^^
    3 │ 	<h1>
    4 │ 		<TextWrapper aria-hidden />
  
  i All headings on a page should have content that is accessible to screen readers.
  

```

```
invalid.jsx:9:2 lint/nursery/useHeadingContent ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ! Provide screen reader accessible content when using heading  elements.
  
     7 │ 		<div aria-hidden />
     8 │ 	</h1>
   > 9 │ 	<h1></h1>
       │ 	^^^^^^^^^
    10 │ </>;
    11 │ 
  
  i All headings on a page should have content that is accessible to screen readers.
  

```


